db/migrate/20230926160040_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :bio
+      t.integer :postCounter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Kindly make sure to add the `Photo` column in the `User` table in order to match the project requirement.

Reply:
Photo column added